server/server.js

```diff
@@ -0,0 +1,44 @@
+require('dotenv').config()
+
+const express          = require('express'),
+      port             = process.env.PORT,
+      path             = require('path'),
+      webpack          = require('webpack'),
+      config           = require('../webpack.config'),
+      compiler         = webpack(config),
```

Comment:
hmm imho lepiej nie mieszać importów z innymi constami/zmiennymi

Reply:
myslalem, zeby je poukladac, ale potem mi to wypadlo z glowy nie pamietam czemu. done